helm/minio-operator/templates/tenant.yaml

```diff
@@ -37,6 +37,7 @@ spec:
           resources:
             requests:
               storage: {{ .size }}
+  {{ end }}
```

Comment:
This is incorrect @gruppferi . All the fields e.g. `tolerations`, `nodeSelector` etc are under the pool section https://github.com/minio/operator/blob/master/examples/tenant.yaml#L59-L129 .

Reply:
@nitisht  then values.yml is wrong, you see they are defined under tenants not under pool. either/or one of them need to be changed. for me it makes sense that they will be defined under but in general one of them need to be changed other wise it is not working if you overwrite any of them

```
tenants:
  # Tenant name
  - name: minio1
    ## Registry location and Tag to download MinIO Server image
    image:
      repository: minio/minio
      tag: RELEASE.2021-03-01T04-20-55Z
      pullPolicy: IfNotPresent
    imagePullSecrets: []
    ## If a scheduler is specified here, Tenant pods will be dispatched by specified scheduler.
    ## If not specified, the Tenant pods will be dispatched by default scheduler.
    scheduler: {}
    ## Used to specify a toleration for a pod
    tolerations: {}
      ## nodeSelector parameters for MinIO Pods. It specifies a map of key-value pairs. For the pod to be
      ## eligible to run on a node, the node must have each of the
    ## indicated key-value pairs as labels.
    ## Read more here: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
    nodeSelector: {}
    ## Affinity settings for MinIO pods. Read more about affinity
    ## here: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity.
    affinity: {}
    ## Configure resource requests and limits for MinIO containers
    resources: {}
    ## Configure security context
    securityContext: {}
```

Reply:
You're right. Can you fix the `values.yaml` in this PR, we can take it in.